src/components/Tooltip.scss

```diff
@@ -0,0 +1,53 @@
+@import "../config/index.scss";
+
+//------------------------------------------------------------//
+
+// * Tooltip Utility
+// * 1. Base CSS
+// * 2. Text & Icon Modifiers
+
+// TODO: add position Modifiers
+// TODO: add js function to automatically update position
+
+//------------------------------------------------------------//
+
+// * 1. Base CSS
+
+.Tooltip {
+  position: relative;
+  //cursor: help;
```

Comment:
What do you think about a `cursor: pointer` here? I think that would provide an indication that the content is interactive.

Reply:
I went away from that because it's not always on a clickable element, and I've always thought of the pointer as defining click ability specifically. But I'm ok with that, especially with the addition of the focus state option.